ecs_types.h

```diff
@@ -17,17 +17,38 @@ template <typename T>
 struct godex_has_get_spawners<T, decltype(void(std::declval<T &>().get_spawners()))> : std::true_type {};
 
 template <typename T, typename = void>
-struct godex_has_bind_methods : std::false_type {};
+struct godex_has__bind_methods : std::false_type {};
 
 template <typename T>
-struct godex_has_bind_methods<T, decltype(void(std::declval<T &>()._bind_methods()))> : std::true_type {};
+struct godex_has__bind_methods<T, decltype(void(std::declval<T &>()._bind_methods()))> : std::true_type {};
+
+template <typename T, typename = void>
+struct godex_has__get : std::false_type {};
+
+template <typename T>
+struct godex_has__get<T, decltype(void(std::declval<T &>()._get(std::declval<const StringName &>(), std::declval<Variant &>())))> : std::true_type {};
+
+template <typename T, typename = void>
+struct godex_has__set : std::false_type {};
+
+template <typename T>
+struct godex_has__set<T, decltype(void(std::declval<T &>()._set(std::declval<const StringName &>(), std::declval<const Variant &>())))> : std::true_type {};
+
+template <typename T, typename = void>
+struct godex_has__get_property_list : std::false_type {};
+
+template <typename T>
+struct godex_has__get_property_list<T, decltype(void(std::declval<T &>()._get_property_list(std::declval<List<PropertyInfo> *>())))> : std::true_type {};
 
 template <typename T, typename = void>
 struct godex_has_storage_config : std::false_type {};
 
 template <typename T>
 struct godex_has_storage_config<T, decltype(void(std::declval<T &>()._get_storage_config(std::declval<Dictionary &>())))> : std::true_type {};
 
+template <typename T, typename = void>
+struct godex_has_is_event : std::false_type {};
```

Comment:
I've just removed this code, which was just some left over, in this PR: https://github.com/GodotECS/godex/pull/238. I think you added it back for mistake, during a rebase, you can just remove it.

Reply:
my bad!